test/features/install_managementconsole.feature

```diff
@@ -0,0 +1,18 @@
+@managementconsole
+Feature: Install Kogito Management Console
+
+  Background:
+    Given Namespace is created
+    And Kogito Operator is deployed with Infinispan and Kafka operators
+
+  Scenario Outline: Install Kogito Management Console
+    Given "<installer>" install Kogito Data Index with 1 replicas
```

Comment:
Please rebase to latest version, the step syntax for component installation was updated, there is no `<installer>` declared in step. It is resolved later in framework.

Reply:
Fixed